website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -37,6 +37,144 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
+|--|--|--|
+| Node | karpenter_nodes_termination_time_seconds | karpenter_nodes_termination_duration_seconds |
+| Node | karpenter_nodes_terminated | karpenter_nodes_terminated_total |
+| Node | karpenter_nodes_leases_deleted | karpenter_nodes_leases_deleted_total |
+| Node | karpenter_nodes_created | karpenter_nodes_created_total |
+| Pod | karpenter_pods_startup_time_seconds | karpenter_pods_startup_duration_seconds |
+| Disruption | karpenter_disruption_replacement_nodeclaim_failures_total | karpenter_voluntary_disruption_queue_failures_total |
+| Disruption | karpenter_disruption_evaluation_duration_seconds | karpenter_voluntary_disruption_decision_evaluation_duration_seconds |
+| Disruption | karpenter_disruption_eligible_nodes | karpenter_voluntary_disruption_eligible_nodes |
+| Disruption | karpenter_disruption_consolidation_timeouts_total | karpenter_voluntary_disruption_consolidation_timeouts_total |
+| Disruption | karpenter_disruption_budgets_allowed_disruptions | karpenter_nodepools_allowed_disruptions |
+| Disruption | karpenter_disruption_actions_performed_total | karpenter_voluntary_disruption_decisions_total |
+| Provisioner | karpenter_provisioner_scheduling_simulation_duration_seconds | karpenter_scheduler_scheduling_duration_seconds |
+| Provisioner | karpenter_provisioner_scheduling_queue_depth | karpenter_scheduler_queue_depth |
+| Interruption | karpenter_interruption_received_messages | karpenter_interruption_received_messages_total |
+| Interruption | karpenter_interruption_deleted_messages | karpenter_interruption_deleted_messages_total |
+| Interruption | karpenter_interruption_message_latency_time_seconds | karpenter_interruption_message_queue_duration_seconds |
+| NodePool     | karpenter_nodepool_usage | karpenter_nodepools_usage |
+| NodePool     | karpenter_nodepool_limit | karpenter_nodepools_limit |
+| NodeClaim    | karpenter_nodeclaims_terminated | karpenter_nodeclaims_terminated_total |
+| NodeClaim    | karpenter_nodeclaims_disrupted | karpenter_nodeclaims_disrupted_total |
+| NodeClaim    | karpenter_nodeclaims_created | karpenter_nodeclaims_created_total |
+
+This table shows v1beta1 metrics that were dropped for v1:
+
+| Metric type | Metric dropped for v1 |
+| Disruption  | karpenter_disruption_replacement_nodeclaim_initialized_seconds |
+| Disruption  | karpenter_disruption_queue_depth |
+| Disruption  | karpenter_disruption_pods_disrupted_total |
+|             | karpenter_consistency_errors |
+| NodeClaim   | karpenter_nodeclaims_registered |
+| NodeClaim   | karpenter_nodeclaims_launched |
+| NodeClaim   | karpenter_nodeclaims_initialized |
+| NodeClaim   | karpenter_nodeclaims_drifted |
+| Provisioner | karpenter_provisioner_scheduling_duration_seconds |
+| Interruption | karpenter_interruption_actions_performed |
+
+#### Results
+
+When you upgrade to `1.0.0` CRDs, Karpenter supports serving both the `v1beta1` versions and the `v1` versions of NodePools, NodeClaims, and EC2NodeClasses.
```

Comment:
Should we ref out to the K8s docs on what serving versions mean as well as what versions for CRDs generally mean, in case users aren't familiar